internal/lsp/opa/scanner/scanner.go

```diff
@@ -0,0 +1,444 @@
+// Copyright 2020 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
```

Comment:
[nit] Lets reference where it came from, it'll make it easier to adjust this in the future

Reply:
There aren't that many scanner.go files in OPA :) But sure, I'll add a link.

Reply:
Commit message would be fine, too!